### ISSUE TITLE
Size of the screenshot for the failed steps should be attached in the html-report in much bigger

### ISSUE THREAD

**Issue by osandadeshan** [getgauge/html-report#226]
### Expected behavior
Screenshot should be much bigger than now.
![image](https://user-images.githubusercontent.com/9147189/57967071-12cd6d80-7978-11e9-8e9a-f51b33cbef0b.png)

### Actual behavior
Screenshot is showing in very small size
![image](https://user-images.githubusercontent.com/9147189/57967035-81f69200-7977-11e9-94cf-078c0827591d.png)
When clicks on it. It will show in full size, but the screenshot attachment in the html report preview should be much bigger. 

### Steps to reproduce
1. Clone the gauge selenium maven java example
2. Change an assertion to fail the scenario
3. Run the spec
4. Open the html-report generated after the execution

### Gauge version
```
Gauge version: 1.0.4
Commit Hash: 3a9a647

Plugins

flash (0.0.1)
html-report (4.0.6)
java (0.7.2.nightly-2019-03-07)
js (2.3.2)
json-report (0.2.2)
python (0.3.4)
screenshot (0.0.1)
spectacle (0.1.3)
xml-report (0.2.1)
```

**Comment by BugDiver** [comment]
@osandadeshan Seems like this issue is same as https://github.com/getgauge/html-report/issues/164.
The later one was fixed as part of release 4.0.7. I see that your html-report version is 4.0.6.
Could you please try to install the latest version of the html-report and check if the issue still exists?

**Comment by osandadeshan** [comment]
![image](https://user-images.githubusercontent.com/9147189/58401494-a2130900-807b-11e9-94d8-14a454ef7681.png)
@BugDiver Cool. This is great. Thanks Vinay I have updated html-report to 4.0.8
